core/templates/components/summary-tile/exploration-summary-tile.component.html

```diff
@@ -118,6 +118,12 @@ <h3 class="mobile-exploration-title protractor-test-exp-summary-tile-title">
       </div>
     </a>
   </mat-card>
+  <button mat-button class="oppia-learner-confirm-button oppia-return-to-parent-button protractor-test-return-to-parent-button"
+            *ngIf="isRefresherExploration" (click)="loadParentExploration()">
+    <span [innerHTML]="'I18N_PLAYER_RETURN_TO_PARENT' | translate"></span>
```

Comment:
This looks like it doesn't have any html tags in the translation. Can you use the translate directive instead of the innerHTML + pipe.